src/pages/venues/[id]/venue.module.scss

```diff
@@ -127,10 +128,6 @@
         margin-bottom: $spacing-2-xl;
       }
     }
-
-    @include breakpoints.respond-above(l) {
-      --spacing-v: #{$spacing-layout-xl};
-    }
```

Comment:
This amount of spacing looked bad when the content was expanded so I removed the extra spacing. I believe that it's in line with the wishes of the designer based on similar changes that were done to the frontpage and search page.